package.json

```diff
@@ -258,6 +258,11 @@
       "type": "object",
       "title": "Puppet",
       "properties": {
+        "puppet.editorService.additionalPuppetSettings": {
```

Comment:
Should this be named `puppetSettings` instead? We know it's already part of the `editorService`, so we don't need the extra information and as a user, it implies to mean there are `normal` or `usual` settings, so I don't know what the difference between those and `additional` is.

Reply:
renamed to `puppetSettings`